adapters/oidc/installed/src/main/java/org/keycloak/adapters/installed/KeycloakInstalled.java

```diff
@@ -161,17 +165,9 @@ public void loginDesktop() throws IOException, VerificationException, OAuthError
 
         String redirectUri = "http://localhost:" + callback.server.getLocalPort();
         String state = UUID.randomUUID().toString();
+        Pkce pkce = generatePkce();
```

Comment:
It seems this always enables pkce without possibility to disable it. I think it can be enabled just if "deployment.isPkce()" is true?

Is it also possible to update docs for KeycloakInstalled adapter and add the info about pkce support?

Reply:
Thanks for the hint! I just added a check for `enable-pkce`